I see the build is failing due to 

```
$ go get -t ./...
# github.com/skycoin/teller/src/exchange
src/exchange/passthrough.go:56:17: cannot use *c2cx.Client literal (type *c2cx.Client) as type "github.com/skycoin/teller/vendor/github.com/skycoin/exchange-api/exchange".Client in field value:
	*c2cx.Client does not implement "github.com/skycoin/teller/vendor/github.com/skycoin/exchange-api/exchange".Client (wrong type for Buy method)
		have Buy(string, "github.com/skycoin/exchange-api/vendor/github.com/shopspring/decimal".Decimal, "github.com/skycoin/exchange-api/vendor/github.com/shopspring/decimal".Decimal) (int, error)
		want Buy(string, "github.com/skycoin/teller/vendor/github.com/shopspring/decimal".Decimal, "github.com/skycoin/teller/vendor/github.com/shopspring/decimal".Decimal) (int, error)
src/exchange/passthrough.go:61:10: cannot use "github.com/skycoin/teller/vendor/github.com/skycoin/exchange-api/exchange".NewTracker() (type "github.com/skycoin/teller/vendor/github.com/skycoin/exchange-api/exchange".Orders) as type "github.com/skycoin/exchange-api/exchange".Orders in field value:
	"github.com/skycoin/teller/vendor/github.com/skycoin/exchange-api/exchange".Orders does not implement "github.com/skycoin/exchange-api/exchange".Orders (wrong type for GetOrderInfo method)
		have GetOrderInfo(int) ("github.com/skycoin/teller/vendor/github.com/skycoin/exchange-api/exchange".Order, error)
		want GetOrderInfo(int) ("github.com/skycoin/exchange-api/exchange".Order, error)
```

Not sure what the best solution is for this.

If `vendor/github.com/skycoin/exchange-api/vendor` exists, we can try removing it and see if that fixes it.

Otherwise we may need to remove the `vendor/` from `exchange-api`.  If we do this, we keep the `Gopkg.toml` file (and maybe the `Gopkg.lock`?) but don't commit the `vendor/` folder. Then add README instructions for fetching dependencies.  Have to check the `dep` docs for the best practices when not committing `vendor/`.  `vendor/` only matters when using the `exchange-api` `cmd/` commands, which isn't important (nobody's using them) so we can require them to sync `vendor/` themselves (even add it as part of the `Makefile`)

--- a/src/config/config.go
+++ b/src/config/config.go
@@ -199,19 +199,19 @@ func (c SkyExchanger) validateWallet() []error {
 
 // ExchangeClient config for the C2CX implementation from skycoin/exchange-api
 type ExchangeClient struct {
-	Key                      string `mapstructure:"key"`
-	Secret                   string `mapstructure:"secret"`
-	OrdersRefreshInterval    uint   `mapstructure:"orders_refresh_interval"`
-	OrderbookRefreshInterval uint   `mapstructure:"orderbook_refresh_interval"`
+	Key                      string        `mapstructure:"key"`
+	Secret                   string        `mapstructure:"secret"`
+	OrdersRefreshInterval    time.Duration `mapstructure:"orders_refresh_interval"`
+	OrderbookRefreshInterval time.Duration `mapstructure:"orderbook_refresh_interval"`
 }
 
 // Validate checks that the refresh intervals are non-zero
 func (ec ExchangeClient) Validate() error {
-	if ec.OrdersRefreshInterval == 0 {
+	if ec.OrdersRefreshInterval <= 0 {
 		return errors.New("OrdersRefreshInterval must be greater than zero")
 	}
 
-	if ec.OrderbookRefreshInterval == 0 {
+	if ec.OrderbookRefreshInterval <= 0 {
 		return errors.New("OrderbookRefreshInterval must be greater than zero")
 	}
 

--- a/src/exchange/passthrough.go
+++ b/src/exchange/passthrough.go
@@ -56,8 +56,8 @@ func NewPassthrough(log logrus.FieldLogger, cfg config.SkyExchanger, store Store
 		exchangeClient: &c2cx.Client{
 			Key:                      cfg.ExchangeClient.Key,
 			Secret:                   cfg.ExchangeClient.Secret,
-			OrdersRefreshInterval:    time.Duration(cfg.ExchangeClient.OrdersRefreshInterval),
-			OrderbookRefreshInterval: time.Duration(cfg.ExchangeClient.OrderbookRefreshInterval),
+			OrdersRefreshInterval:    cfg.ExchangeClient.OrdersRefreshInterval,
+			OrderbookRefreshInterval: cfg.ExchangeClient.OrderbookRefreshInterval,
 			Orders:     exchange.NewTracker(),
 			Orderbooks: orderbookDatabase,
 		},

